Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue2414.cs

```diff
@@ -63,15 +63,14 @@ public void TestDoesntCrashShowingContextMenu ()
 			var screenBounds = RunningApp.Query (q => q.Raw ("* index:0"))[0].Rect;
 
 			var cell = RunningApp.Query (c => c.Marked ("Swipe ME")) [0];
-			if (RunningApp is iOSApp) {
-				RunningApp.DragCoordinates (screenBounds.Width - 10, cell.Rect.CenterY, 0, cell.Rect.CenterY);
-				//TODO: fix this when context menu bug is fixed
-				RunningApp.WaitForElement (c => c.Marked ("Text4"));
-			}
-			else {
-				RunningApp.TouchAndHoldCoordinates (cell.Rect.CenterX, cell.Rect.CenterY);
-				RunningApp.WaitForElement (c => c.Marked ("Text0"));
-			}
+#if __IOS__
```

Comment:
Ah, wrt my previous comment, I see that it is easier to follow the logic with the ifdefs here. And in the IDE we get a nice graying effect for the other platform. Nice!